src/components/binding/binding-loader.js

```diff
@@ -87,14 +92,14 @@ genotet.BindingLoader.prototype.loadFullTrack = function(trackIndex, fileName,
   };
   this.get(genotet.data.serverURL, params, function(data) {
     var track = {
+      gene: data.gene,
       fileName: fileName,
       overview: data,
       detail: data
     };
-    var addTrack = trackIndex == this.data.tracks.length;
     this.data.tracks[trackIndex] = track;
     this.updateRanges_();
-    if (addTrack) {
+    if (isAddTrack) {
       // Add one more track.
       this.loadBindingList();
```

Comment:
Because we need update the panel when adding a new track, and we should load binding list before we update the panel. BTW, you wrote this before, don't remember? :)